bin/setup.dart

```diff
@@ -107,7 +108,7 @@ void buildWrapper() {
   Directory.current = pwd;
   final moveLocation = '$binaryStorageDir${Platform.operatingSystem}64';
   Directory(moveLocation).createSync(recursive: true);
-  final buildOutputPath = Platform.isLinux
+  final buildOutputPath = !Platform.isWindows
```

Comment:
This might not be the right thing to do with isIOS, isAndroid. Maybe make it a switch statement and/or separate method instead of an inline if? (Same for other inline ifs)